target/linux/lantiq/xway/config-5.4

```diff
@@ -43,3 +43,6 @@ CONFIG_UBIFS_FS_LZO=y
 CONFIG_USB=y
```

Comment:
Please run: ```make kernel_oldconfig CONFIG_TARGET=subtarget```

Reply:
That was what I was looking for, thanks!